dataplicity/m2m/remoteprocess.py

```diff
@@ -71,13 +71,14 @@ def write_master(self, data):
         super(RemoteProcess, self).write_master(data)
 
     def close(self):
-        if not self._closed:
+        if not self._closed and self.pid is not None:
             log.debug('sending kill signal to %r', self)
             # TODO: Implement a non-blocking kill
             os.kill(self.pid, signal.SIGKILL)
+            log.debug('waiting for %r', self)
```

Comment:
Just being paranoid, but I'm fairly certain there is nothing to go wrong with killing a child process.